src/css/styles.css

```diff
@@ -102,16 +102,20 @@ body, html {
 
 .temperature {
     font-weight: bold;
+    position: relative;
 }
 
 .unit {
     font-size: 18px;
-    margin-left: 5px;
+    margin-left: 12px;
+    margin-bottom: 10px;
```

Comment:
Because you've not placed the unit div in the correct element, these stylings are totally wrong.
You should use `top`, `bottom`, `left`, and `right` CSS properties while working with `position: absolute;`